content/en/about/faq/setup/install-method-selection.md

```diff
@@ -23,34 +23,37 @@ The following lists some of the pros and cons of each of the available methods:
     - The `istioctl` command can set values automatically based on your running environment,
       thereby producing varying installations in different Kubernetes environments.
 
-1. [istioctl manifest generate](/docs/setup/install/istioctl/#generate-a-manifest-before-installation)
+1. [Install using Helm](/docs/setup/install/helm/)
 
-    Generate the Kubernetes manifest and then apply with `kubectl apply --prune`.
-    This method is suitable where strict auditing or augmentation of output manifests is needed.
+    Using Helm charts allows easy integration with Helm based workflows and automated resource pruning during upgrades.
 
     Pros:
 
-    - Resources are generated from the same `IstioOperator` API as used in `istioctl install`.
-    - Uses the `IstioOperator` API which provides extensive configuration/customization options.
+    - Familiar approach using industry standard tooling.
+    - Helm native release and upgrade management.
 
     Cons:
 
-    - Some checks performed in `istioctl install` are not done.
-    - UX is less streamlined compared to `istioctl install`.
-    - Error reporting is not as robust as `istioctl install` for the apply step.
+    - Fewer checks and validations compared to `istioctl install`.
+    - Some administrative tasks require more steps and have higher complexity.
+
+1. Generated Kubernetes manifest
 
-1. [Install using Helm](/docs/setup/install/helm/)
+    - [Generating Kubernetes manifests with `istioctl`](/docs/setup/install/istioctl/#generate-a-manifest-before-installation)
+    - [Generating Kubernetes manifests with `helm`](/docs/setup/install/helm/#generate-a-manifest-before-installation)
 
-    Using Helm charts allows easy integration with Helm based workflows and automated resource pruning during upgrades.
+    This method is suitable where strict auditing or augmentation of output manifests is required, or there are 3rd party tooling constraints.
```

Comment:
is constraints too tough a term?

something along the lines of "or there is a requirement to use other installation tooling"

Reply:
If you feel it is I'm happy to change it, I don't care much. Every additional piece of tooling is a constraint if you're an operator and you're reading this.

Reply:
```suggestion
    This method is suitable where strict auditing or augmentation of output manifests is required, or there are third-party tooling constraints.
```

That's the least offensive change 😄